create_framework/dependency_injection.rst

```diff
@@ -25,7 +25,7 @@ to it::
             $argumentResolver = new HttpKernel\Controller\ArgumentResolver();
 
             $dispatcher = new EventDispatcher();
-            $dispatcher->addSubscriber(new HttpKernel\EventListener\RouterListener($matcher));
+            $dispatcher->addSubscriber(new HttpKernel\EventListener\RouterListener($matcher, new RequestStack()));
```

Comment:
Using the same request stack than the parent::__construct seems not the good solution. 
Here, $this->requestStack() is null. So the good solution is to instanciate an independant object